src/app/integrations/edit-page/step-configure/filter-steps/basic-filter.component.ts

```diff
@@ -18,71 +26,115 @@ import { BasicFilter } from './filter.interface';
   selector: 'syndesis-basic-filter',
   templateUrl: './basic-filter.component.html',
   encapsulation: ViewEncapsulation.None,
-  styleUrls: [ './basic-filter.component.scss' ],
+  styleUrls: ['./basic-filter.component.scss'],
 })
-
 export class BasicFilterComponent implements OnChanges {
-
   basicFilterModel: DynamicFormControlModel[];
   formGroup: FormGroup;
   predicateControl: FormControl;
   predicateModel: DynamicInputModel;
   rulesArrayControl: FormArray;
   rulesArrayModel: DynamicFormArrayModel;
+  loading = true;
 
   @Input() position;
   @Input()
   configuredProperties: BasicFilter = {
```

Comment:
Development artifact I guess, from before this was showing data from the integration?  It can come out really...